### PR TITLE
new text

### PR DESCRIPTION
**Pull request by JoannaSliwa**
new text

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# demo
\ No newline at end of file
+# demo
+
+proba 2
```
Comment (JoannaSliwa) [inline README.md:3]: This is the new text

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# demo
\ No newline at end of file
+# demo
```
Comment (JoannaSliwa) [inline README.md:1]: This satyed teh same

File: README.md (hunk 1 above)
Comment (ccalvan) [reply README.md:3]: ok good

File: README.md (hunk 2 above)
Comment (ccalvan) [reply README.md:1]: where

**Pull request merged**